spread.yaml

```diff
@@ -659,29 +659,13 @@ suites:
             TRAVIS_TAG: "$(HOST: echo $TRAVIS_TAG)"
             COVERMODE: "$(HOST: echo $COVERMODE)"
         prepare: |
-            #shellcheck source=tests/lib/prepare.sh
-            . "$TESTSLIB"/prepare.sh
-            prepare_classic
+            "$TESTSLIB"/prepare-restore.sh --prepare-suite
```

Comment:
Why do we even bother to prepare anything for unit tests?

Reply:
In this case "$TESTSLIB"/prepare-restore.sh --prepare-suite and prepare_classic is almost the same, because prepare_classic is the only function called while the suite is prepared for classic.
We changed that to use the same code than the other test suites on prepare.